helm/hpcc/values.schema.json

```diff
@@ -2796,11 +2796,6 @@
           "description": "[Optional] Seconds to keep Thor instance running, waiting for more graphs to execute",
           "minimum": 1
         },
-        "multiJobLinger": {
```

Comment:
Better to mark as deprecated, rather than removing? - otherwise charts with it set to true will not deploy

Reply:
Thor's schema allows arbitrary additional properties, i.e. via:
```
      "additionalProperties": { "type": ["integer", "string", "boolean"] },
```

So it wouldn't actually break anything if it continued to be present.
However, Thor and others (we're not consistent on this front) probably should have a strict schema.
I'll mark as deprecated. Can consider making rest of schema in various areas strict too separately.

Reply:
@ghalliday - restored to schema. Please review.